### PR TITLE
10167: Add aria-label to Estimate Your Benefits CalculatorResultRow

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
While testing the college and OTJ flows, I noticed that iOS VoiceOver was reading the wrong data points back after I made filter changes and pressed the "Update your benefits" button. Screenshot and steps to recreate below.

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/10167)

## Screenshots
N/A

## Acceptance criteria
- [x] iOS VoiceOver reads the correct data before and after I have updated my benefits

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 1]
```diff
@@ -355,13 +355,13 @@
   }
 
   .eyb-button {
-    &.usa-accordion-button[aria-expanded='false'] {
-      background-image: url('/img/arrow-up.png');
-      background-image: url('/img/arrow-up.svg');
+    &.usa-accordion-button[aria-expanded="false"] {
```
Comment (elder-joseph) [inline src/applications/gi/sass/partials/_gi-profile-page.scss:358]: Linting change

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 2]
```diff
@@ -436,4 +436,9 @@
   .secondary-sco-list {
     padding-left: 0;
   }
+
+  .eyb-value-header {
```
Comment (zurbergram) [inline src/applications/gi/sass/partials/_gi-profile-page.scss:440]: can you put this class inside the `.your-estimated-benefits` class?

Comment (elder-joseph) [reply src/applications/gi/sass/partials/_gi-profile-page.scss:440]: Yeah sure, let me move that real quick

File: src/applications/gi/sass/partials/_gi-profile-page.scss [hunk 3]
```diff
@@ -355,13 +359,13 @@
...
+    &.usa-accordion-button[aria-expanded="false"] {
+      background-image: url("/img/arrow-up.png");
+      background-image: url("/img/arrow-up.svg");
     }
-    &.usa-accordion-button[aria-expanded='true'] {
-      background-image: url('/img/arrow-down.png');
-      background-image: url('/img/arrow-down.svg');
+    &.usa-accordion-button[aria-expanded="true"] {
+      background-image: url("/img/arrow-down.png");
+      background-image: url("/img/arrow-down.svg");
```
Comment (shawkey-daniel) [inline src/applications/gi/sass/partials/_gi-profile-page.scss:368]: Can you fix the linting on your system to prefer single quotes?

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]

File: src/applications/gi/sass/partials/_gi-profile-page.scss (hunk 3 above)
Comment (elder-joseph) [reply src/applications/gi/sass/partials/_gi-profile-page.scss:368]: Yeah I can do that. Not ideal having linting changes injected into other changes being made. I will update that for this file as well.

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by sonntag-adam — APPROVED** [review]

**Pull request merged**